src/NDItk/multigroup/Metadata.hpp

```diff
@@ -0,0 +1,159 @@
+#ifndef NJOY_NDITK_MULTIGROUP_METADATA
+#define NJOY_NDITK_MULTIGROUP_METADATA
+
+// system includes
+
+// other includes
+#include "tools/Log.hpp"
+#include "NDItk/base/SingleIntegerRecord.hpp"
+#include "NDItk/base/SingleRealRecord.hpp"
+#include "NDItk/base/SingleStringRecord.hpp"
+
+namespace njoy {
+namespace NDItk {
+namespace multigroup {
+
+/**
+ *  @brief Metadata associated to a neutron NDI file
+ */
+class Metadata {
+
+  /* fields */
+
+  base::SingleStringRecord zaid_;
```

Comment:
The underscore is simply a coding style convention to identify private member variables. You'll sometimes see the underscore in front of the variable name, but we do not use that style since prepending a variable name with underscores is common practice for standard library development.